### PR TITLE
DDF-6074 Fixes query request filters not showing up in the security audit log

### PR DESCRIPTION
**Pull request by blen-desta**
#### What does this PR do?
Fixes query request filters not showing up in the security audit log

#### Who is reviewing it? 
@bakejeyner
@garrettfreibott  
@SmithJosh 
@stustison 

#### Select relevant component teams: 
@codice/security 

#### How should this be tested?
Query Intrigue using different filters and verify that those filters are logged in the security log.

#### What are the relevant tickets?
Fixes: #6074

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/security/catalog-security-logging/src/main/java/org/codice/ddf/catalog/security/logging/SecurityLoggingPlugin.java [hunk 1]
```diff
@@ -112,11 +111,8 @@ public DeleteRequest process(DeleteRequest input)
   @Override
   public QueryRequest process(QueryRequest input)
       throws PluginExecutionException, StopProcessingException {
-    String additional = "";
     Query query = input.getQuery();
-    if (query instanceof QueryImpl) {
-      additional = ((QueryImpl) query).getFilter().toString();
-    }
+    String additional = query.toString();
```
Comment (SmithJosh) [inline catalog/security/catalog-security-logging/src/main/java/org/codice/ddf/catalog/security/logging/SecurityLoggingPlugin.java:115]: ❓ Can query be null?

Comment (stustison) [reply catalog/security/catalog-security-logging/src/main/java/org/codice/ddf/catalog/security/logging/SecurityLoggingPlugin.java:115]: No, I don't think so.

**Comment by blen-desta** [comment]
build now

**Comment by SmithJosh** [comment]
Verified the query filter is included in the security log. 
As an aside, it's actually included 4 times, which seems a little overkill, but it's there!

```
[INFO ] 2020-05-26T11:46:13,472 | tp539405560-1155 | securityLogger  |  Subject: admin Performing QUERY_REQUEST operation QueryImpl{filter=QueryImpl{filter=QueryImpl{filter=[ anyText is like pakistan ], startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@7b765047, sortOrder=SortOrder[DESCENDING]}, properties=null}, startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@7b765047, sortOrder=SortOrder[DESCENDING]}, properties=null}, startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@7b765047, sortOrder=SortOrder[DESCENDING]}, properties=null} on catalog.
[INFO ] 2020-05-26T11:46:13,497 | tp539405560-1155 | securityLogger  |  Subject: admin Performing QUERY_REQUEST operation QueryImpl{filter=[QueryImpl{filter=QueryImpl{filter=QueryImpl{filter=QueryImpl{filter=[ anyText is like pakistan ], startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@7b765047, sortOrder=SortOrder[DESCENDING]}, properties=null}, startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@7b765047, sortOrder=SortOrder[DESCENDING]}, properties=null}, startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@7b765047, sortOrder=SortOrder[DESCENDING]}, properties=null}, startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@7b765047, sortOrder=SortOrder[DESCENDING]}, properties=null} AND [[ validation-errors IS NULL ]]], startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@7b765047, sortOrder=SortOrder[DESCENDING]}, properties=null} for source ddf.distribution on catalog.
[INFO ] 2020-05-26T11:46:13,519 | tp539405560-1148 | securityLogger  |  Subject: admin Performing QUERY_REQUEST operation QueryImpl{filter=QueryImpl{filter=QueryImpl{filter=[[ anyText is like pakistan ] AND [ metacard_source = ddf.distribution ]], startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@6f7016d6, sortOrder=SortOrder[DESCENDING]}, properties=null}, startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@6f7016d6, sortOrder=SortOrder[DESCENDING]}, properties=null}, startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@6f7016d6, sortOrder=SortOrder[DESCENDING]}, properties=null} on catalog.
[INFO ] 2020-05-26T11:46:17,479 | tp539405560-1148 | securityLogger  |  Subject: admin Performing QUERY_REQUEST operation QueryImpl{filter=[QueryImpl{filter=[QueryImpl{filter=QueryImpl{filter=QueryImpl{filter=QueryImpl{filter=[[ anyText is like pakistan ] AND [ metacard_source = ddf.distribution ]], startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@6f7016d6, sortOrder=SortOrder[DESCENDING]}, properties=null}, startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@6f7016d6, sortOrder=SortOrder[DESCENDING]}, properties=null}, startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@6f7016d6, sortOrder=SortOrder[DESCENDING]}, properties=null}, startIndex=1, pageSize=250, requestsTotalResultsCount=true, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@6f7016d6, sortOrder=SortOrder[DESCENDING]}, properties=null} AND [[ metacard_source = ddf.distribution ]]], startIndex=1, pageSize=250, requestsTotalResultsCount=false, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@6f7016d6, sortOrder=SortOrder[DESCENDING]}, properties=null} AND [[ validation-errors IS NULL ]]], startIndex=1, pageSize=250, requestsTotalResultsCount=false, timeoutMillis=300000, sortBy=SortByImpl{propertyName=ddf.catalog.filter.impl.PropertyNameImpl@6f7016d6, sortOrder=SortOrder[DESCENDING]}, properties=null} for source ddf.catalog.cache.solr.impl.SolrCacheSource on catalog.
```

**Pull request merged**